src/guide/migration/transition.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Transition Class Change
+badges:
+  - breaking
+---
+
+# {{ $frontmatter.title }} <MigrationBadges :badges="$frontmatter.badges" />
+
+## Overview
+
+The `v-enter` transition class has been renamed to `v-enter-from` and the `v-leave` transition class is renamed to `v-leave-from`.
```

Comment:
```suggestion
The `v-enter` transition class has been renamed to `v-enter-from` and the `v-leave` transition class to `v-leave-from`.
```

Reply:
```suggestion
The `v-enter` transition class has been renamed to `v-enter-from` and the `v-leave` transition class has been renamed to `v-leave-from`.
```

Reply:
This might be more consistent since without it "the v-leave transition class to v-leave-from" might be a little confusing for non-native English speakers.